mavros_msgs/GlobalPositionTarget target
---
sensor_msgs/NavSatFix fix
